assets/tailwind.config.js

```diff
@@ -18,6 +18,9 @@ module.exports = {
             container: {
                 center: true,
             },
+            margin: {
+            'half-screen': '-50vw',
+            },
```

Comment:
I smell a short blog post :wink:

Reply:
this and friends allow us to break out of the width defined on the `<main>` parent. File this one under "hacks"